### PR TITLE
PT-689 split url

### PR DESCRIPTION
**Pull request by dimon37**
In order to be able to easily updated the DB, datasource URL needs to be split

### CODE REVIEW

File: src/main/resources/application-custom.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ spring:
     check-location: false
     baselineOnMigrate: true
   datasource:
-    url: ${datasource.url}
+    url: jdbc:mysql://${DB_HOST}:${DB_PORT}/${DB_NAME}?${DB_PARAMS}
```
Comment (hankwallace) [inline src/main/resources/application-custom.yaml]: Are all task definitions using `jdbc:mysql://`?

File: src/main/resources/application-custom.yaml [hunk 2]
```diff
@@ -15,7 +15,7 @@ spring:
...
     username: ${datasource.username}
```
Comment (hankwallace) [inline src/main/resources/application-custom.yaml]: Please also change these to:
```
DB_USER
DB_PASSWORD
```

File: src/main/resources/application-custom.yaml (hunk 1 above)
Comment (dimon37) [reply src/main/resources/application-custom.yaml]: yes

Comment (dimon37) [reply src/main/resources/application-custom.yaml]: Do you thin we should allow for PostgresQL? I can easily add DB_PROTOCOL with default value of mysql

File: src/main/resources/application-custom.yaml (hunk 2 above)
Comment (dimon37) [reply src/main/resources/application-custom.yaml]: Should we just remove those completely and simply use SPRING_DATASOURCE_USERNAME and SPRING_DATASOURCE_PASSWORD ?

Comment (hankwallace) [reply src/main/resources/application-custom.yaml]: No, I'd like to be consistent with the other `DB_` environment variable usage.

Comment (dimon37) [reply src/main/resources/application-custom.yaml]: Done

**Review by hankwallace — APPROVED** [review]

**Pull request merged**